### DISCUSSION TITLE
Unable to join Github Student Education

### DISCUSSION THREAD

**Discussion by codeShowREX** [community/community#114747] [GitHub Education] [answered]
### Hi everyone,

I am currently pursuing a Bachelor of Computer Applications (BCA) degree from Maulana Abul Kalam Azad University of Technology, West Bengal, India (MAKAUT). Unfortunately, my university did not provide student email addresses, which are necessary to join the Github Student Education program. I have already raised a ticket regarding this issue and uploaded all the required academic verification documents. However, it has been more than four weeks since I submitted the ticket, and I have not received any response from them yet. Can anyone please help me with what I should do now?

In case it is needed, here is my submitted **Ticket ID: 2608953**.
@jmeridth

**Comment by avni** [comment, marked as answer]
Hi @codeShowREX, our analyst team responded in the ticket, thank you!

**Comment by ettaboyle** [comment]
Hi @codeShowREX,

You tagged a GitHub team member in your post. **Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment.** Please review our [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) for more information.

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.

**Any repeat and future violations may result in a temporary or indefinite block from the Community.** Thanks for understanding.

**Reply by safdethgredd** [reply]
https://wa.me/9647841506157

**Comment by Kaka1411** [comment]
rraah22@gmail.com
